src/main.cpp

```diff
@@ -110,8 +110,10 @@ QCoreApplication * createApplication(int &argc, char *argv[], unsigned int &acti
     if( (action) & ( 1 | 2) )
         return new QCoreApplication(argc, argv);  // Ah, we're gonna bail out early, just need a command-line application
     else {
+#if defined (Q_OS_WIN32)
         // Force OpenGL use as we use some functions that aren't provided by Qt's OpenGL layer on Windows (QOpenGLFunctions)
         QApplication::setAttribute(Qt::AA_UseDesktopOpenGL);
```

Comment:
From reading the documentation on my systems this is good at least as far back as Qt 5.4 but is it okay right the way back to 5.0.2?